sdk/src/main/java/co/omise/android/ui/PaymentCreatorActivity.kt

```diff
@@ -260,15 +262,15 @@ private class PaymentCreatorNavigationImpl(
 
     override fun navigateToInstallmentChooser(allowedInstalls: List<PaymentMethod>) {
         val fragment =
-            InstallmentChooserFragment.newInstance(allowedInstalls).apply {
+            InstallmentChooserFragment.newInstance(allowedInstalls, amount, capability.limits?.installmentAmount?.min ?: 0).apply {
```

Comment:
IMO, it would improve readability if we assign `capability.limits?.installmentAmount?.min ?: 0)` to a variable and use it in the function signature instead.

Reply:
After some investigation it would not be very neat, as we will have to pass some stuff twice to propagate the value as intended if i want to implement you r suggestion in all the similar functions .. Better not do this now.

Reply:
maybe just store the value in a variable inside the function